src/pages/[language]/blog/page/index.query.graphql

```diff
@@ -37,9 +37,24 @@ query Blog(
     slug
   }
   tag: tag(filter: { id: { eq: $tagId } }) {
+    i18nSlugs: _allSlugLocales {
+      locale
+      value
+    }
     id
     title
     slug
+    blogTopicSocial {
```

Comment:
We've avoided using the term 'topic' in the codebase, so maybe we should stick with 'tag'?
```suggestion
    blogTagSocial {
```